src/test/resources/org/powerflows/dmn/engine/configuration/decision-with-literal-expressions-only.yml

```diff
@@ -0,0 +1,37 @@
+id: some_table_id_1
+name: Some Table Name
+hit-policy: UNIQUE
+fields:
+  in:
+    inputOne:
+      description: Some Input 1 Description
+      type: INTEGER
+      expression-type: LITERAL
+    inputTwo:
+      type: INTEGER
+  out:
+    outputOne:
+      description: Some Output 1 Description
+      type: BOOLEAN
+    outputTwo:
+      description: Some Output 2 Description
+      type: STRING
+rules:
+- description: Some Rule 1 Description
+  in:
+    inputOne:
+      expression-type: LITERAL
```

Comment:
literal declarations are redundant, use name: value notation

Reply:
intentional, I wanted to have it described in this way